ai-server/main.py

```
@@ -59,53 +79,114 @@ def health():
+        for i, doc in enumerate(final_docs):
+            vector = embeddings.embed_query(doc.page_content)
+
+            # 헤더 메타데이터(Header 1, Header 2 등) + 문서 식별 메타데이터 병합
+            # ChromaDB는 str/int/float/bool 타입만 허용하므로 필터링한다
+            metadata: dict = {
+                "document_id": str(document_id),
+                "source": filename,
+            }
+            for k, v in doc.metadata.items():
+                if isinstance(v, (str, int, float, bool)):
+                    metadata[k] = v
```

Ruff (0.15.10)

[warning] Use a dictionary comprehension instead of a for-loop

(PERF403)